projects/wave-core-new/src/lib/wave-core.module.ts

```diff
@@ -215,7 +215,7 @@ const WAVE_COMPONENTS = [
 ];
 
 @NgModule({
-    declarations: [...WAVE_PIPES, ...WAVE_COMPONENTS],
+    declarations: [...WAVE_PIPES, ...WAVE_COMPONENTS, NotificationsComponent],
```

Comment:
please add your component to the WAVE_COMPONENTS constant

Reply:
d27ffd9b23a6bb4f382ea9cea14eb83f3fb9ba10